test/test_transforms_v2_refactored.py

```diff
@@ -1634,3 +1636,64 @@ def test_transform_negative_degrees_error(self):
     def test_transform_unknown_fill_error(self):
         with pytest.raises(TypeError, match="Got inappropriate fill arg"):
             transforms.RandomAffine(degrees=0, fill="fill")
+
+
+class TestCompose:
+    class BuiltinTransform(transforms.Transform):
+        def _transform(self, inpt, params):
+            return inpt
+
+    class PackedInputTransform(nn.Module):
+        def forward(self, sample):
+            image, label = sample
+            return image, label
```

Comment:
I understand that both work, but wouldn't this make more sense as

```suggestion
    class PackedInputTransform(nn.Module):
        def forward(self, sample):
            return sample
```
?

returning `image, label` works right now, but IDK if this is something we actually want to enforce support for?

Reply:
errrrrrrrrrrrrrrr OK lol it's the same thing (minor pref for just retuning `sample` but not blocking)